src/test/java/io/xstefank/wildfly/bot/PRAppendingMessage.java

```diff
@@ -0,0 +1,235 @@
+package io.xstefank.wildfly.bot;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTest;
+import io.quarkus.test.junit.QuarkusTest;
+import io.xstefank.wildfly.bot.model.RuntimeConstants;
+import io.xstefank.wildfly.bot.util.DescriptionStringBuilder;
+import io.xstefank.wildfly.bot.utils.Action;
+import io.xstefank.wildfly.bot.utils.GitHubJson;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHEvent;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.given;
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.mockCommits;
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.mockEmptyComments;
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.mockEmptyFileDetails;
+import static io.xstefank.wildfly.bot.helper.MockedGHPullRequestProcessor.processPullRequestMock;
+
+@QuarkusTest
+@GitHubAppTest
+public class PRAppendingMessage {
+
+    private String wildflyConfigFile = """
+            wildfly:
+              format:
+                commit:
+                  enabled: false
+            """;
+
+    private String appendedMessage = String.format("""
+            %s\n
+            > %s%s"""  + RuntimeConstants.BOT_MESSAGE_FOOTER, RuntimeConstants.BOT_MESSAGE_HEADER, RuntimeConstants.BOT_JIRA_LINKS_HEADER, "%s");
```

Comment:
Otherwise we lose the %s, which is used again. This `appendMessage` is further used for `String.format` and the `%s` contains the body for individual tests.